df_py/predictoor/queries.py

```diff
@@ -70,10 +119,18 @@ def query_predictoor_contracts(chain_id: int) -> Dict[str, PredictContract]:
                 continue
 
             nft_addr = contract["token"]["nft"]["id"]
+            info725 = contract["token"]["nft"]["nftData"]
+            info = info_from_725(info725)
+            pair = info["pair"]
+            timeframe = info["timeframe"]
+            source = info["source"]
+
+            asset_name = f"{pair}-{timeframe}-{source}"
```

Comment:
Please make it `f"{pair}-{source}-{timeframe}"`

(It should be like this everywhere, btw. Since pair and source are intrinsically bound to each other.)